pkg/restapi/helper/helper.go

```diff
@@ -0,0 +1,43 @@
+/*
```

Comment:
Note that these were existing functions - I just moved them from start_test.go to a common location since I use them in multiple packages now.

Reply:
This should be under a restapi/internal/<pkgname> path. Are these test utilities for the REST packages?

Reply:
Since I can't move it to an internal package due to it being used in different packages where the parent folders don't share a common path, @sudeshrshetty let me know that you suggested that I should instead duplicated those test utility functions in each test file. That's done.